data/create_xinZhuang_field_data.js

```diff
@@ -0,0 +1,6 @@
+const createData = require('./utils/create-field-data');
+
+const fieldData = require('./field-info/xinZhuang-Zone.json'); // data of field (include field, orientation, level, zone)
+const folder = './data/space-data/xinZhuang';
```

Comment:
resolve to absolute path for compatibility

Reply:
fixed